### PR TITLE
Proxy contracts Tests fixes

### PR DESCRIPTION
**Pull request by alistair-singh**
There is some overlap with changes made in https://github.com/Snowfork/snowbridge/pull/890. But this PR renamed tests and fixes for the XcmConverter tests.

### CODE REVIEW

File: parachain/pallets/control/src/lib.rs [hunk 1]
```diff
@@ -33,7 +33,6 @@ pub mod pallet {
 	use super::*;
 	use frame_support::{pallet_prelude::*, traits::EnsureOrigin};
 	use frame_system::pallet_prelude::*;
-	use xcm::v3::MultiLocation;
```
Comment (doubledup) [inline parachain/pallets/control/src/lib.rs:36]: #890 Removes `xcm::prelude::*` above. If we merge both we'll probably end up with no `MultiLocation` here. Could we instead remove the prelude above?

**Review by doubledup — APPROVED** [review]
One merge-conflict-free error case to watch out for. Otherwise we just need to be careful when merging.

File: parachain/pallets/control/Cargo.toml [hunk 2]
```diff
@@ -50,7 +47,11 @@ std = [
 	"sp-io/std",
 	"sp-runtime/std",
 	"xcm/std",
+	"xcm-builder/std",
```
Comment (yrong) [inline parachain/pallets/control/Cargo.toml:50]: Seems `xcm-builder` is only used in test codes and not requied here.

**Comment by yrong** [comment]
@alistair-singh 
I've merged this PR and my previous #890 together into branch `proxy-contracts`. So could you help to double-check `proxy-contracts` if anything missing or wrong.

**Review by yrong — APPROVED** [review]

File: parachain/pallets/control/Cargo.toml (hunk 2 above)
Comment (alistair-singh) [reply parachain/pallets/control/Cargo.toml:50]: this failed when running `cargo test --features runtime-benchmarks` so I had to add this line.

File: parachain/pallets/control/src/lib.rs (hunk 1 above)
Comment (alistair-singh) [reply parachain/pallets/control/src/lib.rs:34]: #890 is merged and this is rebased on top of that.

File: parachain/pallets/control/Cargo.toml (hunk 2 above)
Comment (yrong) [reply parachain/pallets/control/Cargo.toml:50]: also require remove `default-features = false` in
`xcm-builder = { git = "https://github.com/paritytech/polkadot.git", branch = "master", default-features = false }` above

Already merged into `proxy-contracts` and seems CI tests for `runtime-benchmarks` just works fine.

**Pull request merged**